pkg/plugins/policies/core/rules/rules.go

```diff
@@ -235,6 +278,29 @@ func SubsetFromTags(tags map[string]string) Subset {
 	return subset
 }
 
+type Element map[string]string
+
+func (e Element) WithKeyValue(key, value string) Element {
+	if e == nil {
+		e = Element{}
+	}
+
+	e[key] = value
+	return e
+}
+
+func MeshElement() Element {
+	return Element{}
+}
+
+func MeshServiceElement(name string) Element {
+	return Element{mesh_proto.ServiceTag: name}
+}
+
+func MeshExternalServiceElement(name string) Element {
+	return Element{mesh_proto.ServiceTag: name}
+}
```

Comment:
Because I noticed that we have such old `Subset` codes, so I want to align with it

https://github.com/kumahq/kuma/blob/86593067f05038a1c7f76446a177538f3efff00f/pkg/plugins/policies/core/rules/rules.go#L218-L228